bundles/org.openhab.binding.lgwebos/README.md

```diff
@@ -20,25 +20,34 @@ In combination with the wake on LAN binding this will allow you to start the TV
 
 ## Binding Configuration
 
-The binding has only one configuration parameter, which is only required if the binding cannot automatically detect openHAB's local IP address: 
-
-| Name    | Description                                                          |
-|---------|----------------------------------------------------------------------|
-| localIP | This is the local IP of your openHAB host on the network. (Optional) |
-
-If LocalIP is not set, the binding will use openHAB's primary IP address, which may be configured under network settings.
+The binding has no configuration parameter. 
```

Comment:
Maybe leave out the Binding Configuration section completely, if there is nothing one can do.

Reply:
There used to be a binding config parameter. I thought for users of the 2.4 version this could be helpful.

Reply:
Sounds fair indeed. 👍